perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_optional_login_namespace_mu.java

```diff
@@ -0,0 +1,73 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.ExtSource;
+import cz.metacentrum.perun.core.api.ExtSourcesManager;
+import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.UserExtSource;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Contains login in the MU namespace if it is available, if not the value is null
+ *
+ * @author David Flor
+ */
+public class urn_perun_user_attribute_def_virt_optional_login_namespace_mu extends urn_perun_user_attribute_def_virt_optional_login_namespace {
```

Comment:
We should probably add `@SkipValueCheckDuringDependencyCheck` class annotation, since it is used in the original attribute module for `user:virt:optionalLogin_namespace_mu`. Also it should be used in parent class too, since both classes do not implement checkAttributeSyntax/Semantics() methods and this annotations optimize the whole checking process by skipping them.